### PR TITLE
chore: Bump node 18.18.0

### PR DESCRIPTION
**Pull request by salisdemos**
To Bump Node, we have to bump a bunch of dependencies too. So this PR has to be merged at the same time a new container gets rolled out with a Node-Version "18.18.0" 

The changes can be tested, by Bumping node in Your local container. (login as root (`dp shell --root`) -> `npm install -g n` --> `n 18.18.0` --> `exit` --> `dp shell` --> `yarn; yarn dev:blp` (or any other project) )
You may run into the well knows "string-with"-Problem. That can be fixed by running `yarn upgrade jest`

OR:
You set Your container-Version to 5.2 . That comes with node 18.20.0 (which also should work)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -185,6 +185,6 @@
     "ff esr"
   ],
   "resolutions": {
-    "@mapbox/mapbox-gl-style-spec": "13.14.0"
+    "@mapbox/mapbox-gl-style-spec": "13.23.1"
```
Comment (spiess-demos) [inline package.json:188]: Do we still have to keep this around?

File: config/packages/nelmio_security.yaml [hunk 2]
```diff
@@ -31,7 +31,7 @@ nelmio_security:
             base-uri: '%demosplan_csp_base_uri%'
             form-action: '%demosplan_csp_form_action%'
             worker-src:
-                - 'none'
+                - 'blob:'
```
Comment (spiess-demos) [inline config/packages/nelmio_security.yaml:34]: This change seems to be unrelated to the current PR.

Comment (salisdemos) [reply config/packages/nelmio_security.yaml:34]: actually no. Bumping the masterPortalApi made this change necessary. So its the last snippet of the change-reaction while "you not simply bump node"

File: package.json (hunk 1 above)
Comment (salisdemos) [reply package.json:188]: what do You mean? If we can drop the whole resolution? Maybe. won't test that within that thing here. was ugly enough

Comment (spiess-demos) [reply package.json:188]: yes i meant to drop the resolution but i agree that's not in scope here.

**Review by spiess-demos — APPROVED** [review]
lgtm (code review only)

**Review by hwiem — APPROVED** [review]
Lgtm!

**Comment by salisdemos** [comment]
Sorry. I had to merge main into the branch again (and recreate the yarn.lock-File), which dismissed the Reviews.

**Review by hwiem — APPROVED** [review]
